### PR TITLE
Main

### PR DESCRIPTION
**Pull request by lekegitrepo**
Hi,
 we are looking forward to feedback/review.
Best regards.
Thais & Temitayo.

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Great work team 👍
You sure did nicely on this project. But it's not ready yet, we need to make some changes to be in line with Microverse guidelines.

- [x] Any sections that include text (paragraphs) should be wrapped with a `section` tag and include an `aside` tag for Editor's picks and a Sign-Up button (there are 2 "Editor's picks" in this page, this one lies aside of the main article). 

- [x] All images and videos should be enclosed by a `figure` tag with `figcaption` that gives a description of the embedded image or video. Not only the images from the main article but also the images from 'More in space and cosmos' and 'Editor's picks'.

- [x] Your project needs to have the sections 'More in space and cosmos', 'Editor's picks' and 'Most popular'

- [x] Your footer must have the list links (News, opinion, arts, living, more and subscribe). Use the ul tag. The style must match the original page. It also must include the NY logo(must be a link) and the 'Go to home page' link.

Please kindly make these changes and submit for code review again.

**Comment by Ispirett** [comment]
Hey, folks great work with the changes you are doing a fantastic job :100: . There are just a few changes that Mircoverse requires before this project can be completed.


### html errors
There are a lot of HTML errors on this page use this tool and please correct them https://validator.w3.org/
### Footer
- [x] Links should be organized using the `<ul>` tag and positioned with display: flex or floats

When finished with these changes please submit for another code review you can use this same pull request link.

File: index.html [hunk 1]
```diff
@@ -120,22 +121,57 @@ <h1 class="headline-container">
...
 					<p>Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
-				</div>
-			</div>
-				<figure class="inflation-theory">
-					<img src="images/theory.png" alt="explanations-of-inflation">
-				</figure>
+		</section>
+	</div>
+
+	<figure class="inflation-theory">
```
Comment (Ispirett) [inline index.html:184]: The image in this section is too big you can make it a little smaller.

**Review by Ispirett — CHANGES_REQUESTED** [review]

**Comment by Oghenebrume50** [comment]
Good fix guys, everything works perfectly, I made a little comment below fix that, merge your branches to master and submit the project completion form

File: style.css [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+	display: block;
+}
+.headline-container {
+	margin-left: 20%;
+}
+#content-img {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
```
Comment (Oghenebrume50) [inline style.css:85]: Currently the image is titled to the left of the page add `align-items: center` to this style to center it horizontally

**Pull request merged**